src/scaffold/ScaffoldView.tsx

```diff
@@ -21,33 +21,26 @@ function getRef(uri: Uri) {
   return `${scheme}://${authority}${path}`;
 }
 
-interface LocalState {
-  repoType: string | undefined;
-  template: string | undefined;
-}
-
-const defaultLocalState = { repoType: undefined, template: undefined };
-
 const FirstPage: React.FC<PageProps> = ({ onNext }) => {
   const { state, actions } = useStore();
-  const [localState, setLocalState] = React.useState<LocalState>(defaultLocalState);
-  const availableTemplates = localState.repoType ? state.templates[localState.repoType] : undefined;
-  const canGoNext = localState.template !== undefined;
+  const options = state.options;
+  const availableTemplates = options.repoType ? state.templates[options.repoType] : undefined;
+  const canGoNext = options.template !== '';
   const loading = availableTemplates === undefined;
   const next = () => {
-    if (localState.repoType && localState.template) {
+    if (options.repoType && options.template) {
       onNext({
```

Comment:
Why? If you place the options anyway in the global store then `onNext()`. Change the signature.